### PR TITLE
Add Need some inspiration admin note

### LINKED ISSUE #4200: New notification: Do you need some inspiration?
**Title:** Do you need some inspiration?
**Content:**  Check some of our favorite customer stories from entrepreneurs, agencies, and developers in our global community.
**Segmentation:** 
New users with `setup_client`: `false` on `wcadmin_storeprofiler_store_details_continue`
**Timing:** 5 days after plugin install
**Cadence:** Once
**CTA leads to:** https://woocommerce.com/success-stories/?utm_source=inbox
**CTA label:** See success stories

### PR DESCRIPTION
**Pull request by becdetat**
Fixes #4200

This adds a new admin note ('Need some inspiration') five days after activation.

### Detailed test instructions:

- note that the WCAdmin plugin needs to have been installed for at least 2 days
- Run the wc-admin-daily cron task
- The below note should appear in the inbox:

image

### Changelog Note:

Enhancement: Add Need some inspiration admin note

### CODE REVIEW

File: src/Notes/WC_Admin_Notes_Need_Some_Inspiration.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_NAME = 'wc-admin-need-some-inspiration';
+
+	/**
+	 * Get the note.
+	 */
+	public static function get_note() {
+		// We want to show the note after five days.
+		if ( ! self::wc_admin_active_for( 5 * DAY_IN_SECONDS ) ) {
```
[the remark this answers is not in the document]
Comment (becdetat) [reply src/Notes/WC_Admin_Notes_Need_Some_Inspiration.php:33]: Good call

**Review by timmyc — APPROVED** [review]
This is testing out well for me in both scenarios:

- New site, note not being shown
- Established site, all the inspiration

I'm fine with this landing without the upper limit myself... seems like everyone could use a little more inspiration these days <3 But cc @pmcpinto for a final call on that.

**Comment by timmyc** [comment]
After a rebase, and Pedro's final call, I'm good with this landing. Thanks Bec!

**Comment by becdetat** [comment]
Whoops, I pressed the button too soon. If @pmcpinto wants the upper limit it should be a simple fix though.

**Comment by pmcpinto** [comment]
> I pressed the button too soon. If @pmcpinto wants the upper limit it should be a simple fix though.

No worries :) We can change the upper limit in the upcoming cycle 👇 

> Usually it takes ~1 month to get the first order, so I think that having 30 days as maximum store age is a good ballpark.

**Pull request merged**